package.json

```diff
@@ -37,11 +37,11 @@
         "prettier": "npx prettier --check .",
         "prettier:fix": "npm run prettier -- --write",
         "prepublishOnly": "npm run release",
-        "preinteg:browser": "npm run build:dev",
-        "integ:browser": "testcafe --config-file=./.testcafe.browser.js --app \"http-server ./build/dev -s\"",
-        "integ:browser:ie": "testcafe ie --app \"http-server ./build/dev -s\" src/__integ__/unsupportedBrowser.test.ts",
-        "preinteg:headless": "npm run build:dev",
-        "integ:headless": "testcafe --config-file=./.testcafe.headless.js --app \"http-server ./build/dev -s\"",
+        "preinteg": "npm run build:dev",
```

Comment:
Doesn't your previous PR address these changes already?

Reply:
Nope.

This gets rid of the duplicate `pre<x>` steps and also allows for you to pass a browser (in main if you try, it first read the config file, which then errors out)

so now both headless and browser integ scripts inherit from one point